Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/model/settings/Setting.java

```diff
@@ -0,0 +1,25 @@
+package org.dolphinemu.dolphinemu.model.settings;
+
+public abstract class Setting
```

Comment:
Class and methods should have documentation, since it's a class that's inherited from.